### PR TITLE
version 18.02

### PR DESCRIPTION
**Pull request by yarl**

### CODE REVIEW

**Comment by Abbe98** [comment]
May I suggest timing the release with the deployment of STL support on Commons(`T184728`)?

I'm planing some minor changes to logging, as well as adding a link to documentation somewhere(over the weekend or early next week).

File: src/pattypan/panes/StartPane.java [hunk 1]
```diff
@@ -112,7 +116,9 @@ private GridPane setContent() {
     String year = new SimpleDateFormat("yyyy").format(new Date());
     this.addRow(40, new WikiLabel(year + " // Pawel Marynowski").setClass("muted"));
 
-    TextFlow flow = new TextFlow(new Text(Util.text("start-bug-found")), bugLink, new Text("("), logFile, new Text(")"));
+    TextFlow flow = new TextFlow(
+            new Text(Util.text("start-bug-found")), bugLink,
+            new Text(" • "), logFile);
```
Comment (Abbe98) [inline src/pattypan/panes/StartPane.java]: Looks a lot better 👍

**Comment by yarl** [comment]
@Abbe98 sure, no rush on that one.

**Comment by yarl** [comment]
@Ambrosiani you are referring to row 124 in which file?
@Abbe98 it looks STL will not be released this month, are you ok with release what we have now?

**Comment by Ambrosiani** [comment]
@yarl "departament" should be "department" here as well: https://github.com/yarl/pattypan/blob/9836af723170d50e6ce6bd3e4b668030ebab8fce/src/pattypan/Settings.java#L124

**Comment by Abbe98** [comment]
@yarl do you think the fixes here is enough to motivate a release? I have some work done on improving the logger but can't promise I will have the time in the coming week to test it enough.

**Pull request merged**